cmd/geth/config.go

```diff
@@ -135,6 +138,7 @@ func makeConfigNode(ctx *cli.Context) (*node.Node, gethConfig) {
 
 	utils.SetShhConfig(ctx, stack, &cfg.Shh)
 
+	utils.SetDashboardConfig(ctx, &cfg.Dashboard)
```

Comment:
Nitpick, move this line one line up (so the empty line is before the `return`). It makes the code cleaner to read.